bundles/SimpleBackendSearchBundle/src/Controller/SearchController.php

```diff
@@ -44,7 +45,134 @@ class SearchController extends AdminController
     use AdminStyleTrait;
 
     /**
-     * @Route("/find", name="pimcore_admin_searchadmin_search_find", methods={"GET", "POST"})
+     * @Route("/quicksearch-get-by-id", name="pimcore_bundle_search_search_quicksearch_by_id", methods={"GET"})
```

Comment:
why the order of methods was changed? ... same order would be easier to review...

Reply:
reordered action 524fa1efda5f8c4a3511e40d19c035c4620f4311